documentation/airport-guides/eggw.md

```diff
@@ -0,0 +1,109 @@
+
+4,538,813 passengers handled in 2015-16
+85,169 aircraft movements in 2015-16
+
+09/27 1,508m, Asphalt
+
+##### London Stansted Airport, EGSS
+London Stansted Airport is an international airport located at Stansted Mountfitchet, Essex. It is 48km northeast of Central London. The airport servers over 170 destinations across Europe and is a base for many low-cost airlines, most notably Ryanair. The airport is owned by the Manchester Airports Group, which operates Manchester Airport and East Midlands Airport. The airport is a focus city for easyJet, Jet2.com, Ryaniar, Thomas Cock and TUI.
+
+24,320,071 passengers handled in 2015-16
+180,430 aircraft movements in 2015-16
+
+04/22 3,049m, Grooved asphalt
+
+##### London Southend Airport, EGMC
+London Southend Airport is an international airport 68km from Central London. The airport serves very few airlines, however it has been increasing passenger numbers over the years. British low-cost carriers Flybe and easyJet have been one of the reasons behind this increase in passenger numbers as they have been starting new routes from Southend recently.
+
+05/23 1,856m, Asphalt
+
+##### London Farnborough Airport, EGLF
+Farnborough Airport is a business and general aviation airport in Hampshire. The airfield is the home of the Farnborough Airshow which is held every two years.
+
+06/2 2,440m, Asphalt
+
+##### London Biggin Hill Airport, EGKB
+Biggin Hill Airport is a general aviation airport 22km south-east of Central London. It was formerly an RAF station. Radar air traffic control services are given by Thames Radar at the LTCC and VFR ATC are provided by the airport itself. There are no commercial air services to or from Biggin Hill; all requests for such commercial services have been rejected by the council and no scheduled nor fare-paying passenger services are permitted to operate here.
+
+03/21 1,820m, Tarmac
+
+
+### Videomap
+Provided with the videomap is a generic videomap for the London Terminal Manoeuvring Area. The extended centrelines and runways of all major airports (Heathrow, Gatwick, Stansted, Luton and City) are displayed, along with the positions of important VORs. CTAs and ATZs of all the major airports are displayed too.
+
+Every dash on an extended centreline represents one (1) nautical mile, with each small 'T' representing four (4) miles.
+All sector splits of the LTMA are drawn on too.
+
+### Additional Resources
+[NATS London TMA Diagram](http://www.nats-uk.ead-it.com/public/index.php%3Foption=com_content&task=blogcategory&id=4&Itemid=11.html)  

```

Comment:
Same comment as above, would this link be better?
http://www.ead.eurocontrol.int/eadbasic/pamslight-9523BF98BD8ED45E830453089C9923C6/7FE5QZZF3FXUS/EN/Charts/ENR/AIRAC/EG_ENR_6_2_1_11_en_2018-02-01.pdf